zappa/handler.py

```diff
@@ -336,9 +336,24 @@ def handler(self, event, context):
 
         # This is a direct command invocation.
         elif event.get('command', None):
-
             whole_function = event['command']
+            args = []
+            kwargs = {}
```

Comment:
I think I'd recommend not going the non-standard way of parsing command line keyword arguments. For most command lines, I'd expect something like `zappa invoke "myapp.myfunc" arg1 arg2 --arg3 foo` to provide args as `['arg1', 'arg2']` and kwargs as `{'arg3': 'foo'}`.

Of course this is up to you @xuru and @Miserlou, but I think leaving out kwargs for the first iteration would be best.